src/debug/di/shimcallback.cpp

```diff
@@ -1402,10 +1402,22 @@ HRESULT ShimProxyCallback::DataBreakpoint(ICorDebugProcess* pProcess, ICorDebugT
         {
             this->m_pProcess.Assign(pProcess);
             this->m_pThread.Assign(pThread);
-            this->m_pContext = pContext;
+
+            this->m_pContext = new BYTE[contextSize];
```

Comment:
I am wondering about OOM handling here, we can't return anything in a constructor.
How about using a CONTEXT field instead?

Reply:
should I assert that contextSize == sizeof(CONTEXT)?